KinConnections/resources/models.py

```diff
@@ -0,0 +1,24 @@
+from flask_marshmallow import Marshmallow
+from marshmallow import Schema, fields, pre_load, validate
+
+ma = Marshmallow()
+
+# {	
+# 	"first_name" : "Fahim",
+# 	"last_name" : "Karim",
+# 	"email" : "fahimkarim@gmail.com",
+# 	"dob" : "2021-08-02",
+# 	"country_origin" : "United States of America",
+# 	"region_current" : "United States of America",
+# 	"gender" : "Male",
+# 	"languages" : ["English"]
+# }
```

Comment:
Something to consider: Are connectors going to be logging in? If so, maybe we should add an `activated` field for the connectors as well? That could help us keep track of first-time logins and prompt them too create a password for themselves.